stablehlo/tests/interpret/composite.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
+
+func.func @add(%arg0: tensor<i64>, %arg1: tensor<i64>) -> tensor<i64> {
+  %0 = stablehlo.add %arg0, %arg1 : tensor<i64>
+  func.return %0 : tensor<i64>
+}
+
+func.func @main() {
```

Comment:
For the sake of using attrs / having a more "custom" composite, lets make this an AddNOp:

```
func.func @add_n.impl(%arg0: tensor<i64>) -> tensor<i64> {
  %0 = stablehlo.constant dense<2> : tensor<i64>
  %1 = stablehlo.add %arg0, %0 : tensor<i64>
  func.return %1 : tensor<i64>
}

func.func @main() {
  %0 = stablehlo.constant dense<1> : tensor<i64>
  %1 = stablehlo.composite "stablehlo.add_n" %0 {
    composite_attributes = { n = 2 : i64 },
    decomposition = @add_n.impl
  } : (tensor<i64>, tensor<i64>) -> tensor<i64>
  check.expect_eq_const %1, dense<3> : tensor<i64>
  func.return
}
```